### DISCUSSION TITLE
Accessibility statement and privacy notice

### DISCUSSION THREAD

**Discussion by kelliedesigner** [ministryofjustice/moj-frontend#270] [EXPERIMENTAL: Components, pages and patterns]
Teams regularly ask each other for their accessibility statement and privacy notice as reference for their current project. 

The MOJ Forms team provides a template accessibility statement and privacy notice:

https://moj-online.service.justice.gov.uk/accessibility/#statement
https://moj-online.service.justice.gov.uk/data-protection/#privacy-notice

Should we add these as patterns in the Pattern Library so their existence is more known across MOJ and can be reused?

**Comment by liammcmurray** [comment]
This is a great idea. Also, reworking those from PDFs into page templates would be the responsible thing to do :)
I'm really surprised these aren't part of wider gov design system, like https://design-system.service.gov.uk/patterns/cookies-page/
Can there be that much variation that a template isn't feasible?
